apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/CreateOrder/CreateOrder.tsx

```diff
@@ -34,6 +34,7 @@ export const CreateOrder = () => {
   const handleSelectOrderType = (index: number) => {
     orderTypeChanged(index === 0 ? 'market' : 'limit')
   }
+  const disabled = createOrder.amounts.buy.quantity === '0' || createOrder.amounts.sell.quantity === '0'
```

Comment:
There is a helper `Quantities.isZero()`
```suggestion
  const disabled = createOrder.amounts.buy.quantity === '0' || createOrder.amounts.sell.quantity === '0'
```